lib/LMSManagers/LMSCustomerManager.php

```diff
@@ -1993,7 +2002,7 @@ public function customerUpdate($customerdata)
         }
 
         $args = array(
-            'extid'          => $customerdata['extid'],
+            'extid'          => empty($customerdata['extid']) ? '' : $customerdata['extid'],
```

Comment:
Czyżbyś tu coś na boku jeszcze dodatkowo  poprawiał?

Reply:
Tak - PHP Warning mi gdzieś się w logu pojawił. Może to tu zostać czy usunąć?